_sources/SequenceMutation/AppendversusConcatenate.rst

```diff
@@ -0,0 +1,74 @@
+..  Copyright (C)  Brad Miller, David Ranum, Jeffrey Elkner, Peter Wentworth, Allen B. Downey, Chris
+    Meyers, and Dario Mitchell.  Permission is granted to copy, distribute
+    and/or modify this document under the terms of the GNU Free Documentation
+    License, Version 1.3 or any later version published by the Free Software
+    Foundation; with Invariant Sections being Forward, Prefaces, and
+    Contributor List, no Front-Cover Texts, and no Back-Cover Texts.  A copy of
+    the license is included in the section entitled "GNU Free Documentation
+    License".
+
+.. qnum::
+   :prefix: seqmut-7-
+   :start: 1
+
+Append versus Concatenate
+-------------------------
+
+The ``append`` method adds a new item to the end of a list.  It is also possible to add a new item to the end of a list by using the concatenation operator.  However, you need to be careful.
+
+Consider the following example.  The original list has 3 integers.  We want to add the word "cat" to the end of the list.
```

Comment:
I Suggest that we introduce the id function here (or earlier when we talk about ``is``) and then print the id of origlist.  Otherwise this example is not so clear because the result looks the same, but the point is that a new object is created when you concatenate.